test/benchmark.js

```diff
@@ -0,0 +1,110 @@
+
+var fs = require('fs');
+var path = require('path');
+var child_process = require('child_process')
```

Comment:
As far as I can tell, this isn't actually used in this file? Unless I missed something.